sni/templates/docs/shelling-out.html

```diff
@@ -465,7 +465,7 @@ <h2 id="the-spoils-of-war">The Spoils of War</h2>
+    Victory in war was sometimes followed by an immediate payment from the losers to the winners. Often this just took the form of looting by the enthusiastic victors, while the losers desperately hid their collectibles. More often, tribute was demanded on a regular basis. In this case, the triple coincidence could and sometimes was avoided by a sophisticated schedule of payments in kind that matched the losing tribe's ability to supply a good or service with the victor's demand for it. However, even with this solution primitive money could provide a better way &ndash; a common medium of value that greatly simplified the terms of payment – very important in an era when terms of the treaty could not be recorded but had to be memorized. In some cases, as with the wampum as used in the Iriquois Confederacy, the collectibles doubled as a primitive mnemonic device that, while not verbatim, could be used as an aid to recall the terms of the treaty. For the winners, collectibles provided a way to collect tribute at closer to the Laffer optimum. For the losers, collectibles buried in caches provided a way to "under-report", leading the victors to believe the losers were less wealthy and thus demand less than they might. Caches of collectibles also provided insurance against over-zealous tribute collectors. Much of the wealth in primitive societies escaped the notice of the missionaries and anthropologists due to its highly secretive nature. Only archeology can reveal the existence of this hidden wealth.
```

Comment:
Victory in war was sometimes followed by an immediate payment from the losers to the ~~victims~~ **winners**